trunk/NDHMS/Routing/Reservoirs/Persistence_Level_Pool_Hybrid/module_persistence_levelpool_hybrid_parameters.F

```diff
@@ -0,0 +1,105 @@
+! This module defines and instantiates objects
+! for a hybrid type reservoir's
+! parameters/properties. Parameters holds
+! static/unchanging variables that are
+! set when the given reservoir object is
+! initialized/instantiated.
+
+#define seconds_in_day 86400
+
+module module_persistence_levelpool_hybrid_parameters
+
+    use module_reservoir_utilities
+    use module_reservoir_base
+    implicit none
+
+    ! Extend/derive hybrid parameters from the abstract base
+    ! struct for reservoir properties.
+    type, extends(reservoir_base_properties_struct) :: hybrid_parameters_interface
```

Comment:
similar to state, 

why not `type, extends(levelpool_parameters_interface) :: levelpool_hybrid_parameters` ?

Reply:
@jdmattern-noaa fixing. Changed in same manner as state.